packages/es-components/src/components/controls/answer-group/AnswerGroup.md

```diff
@@ -0,0 +1,58 @@
+Group `RadioButton` components together using a `RadioGroup`. Any additional props passed to `RadioGroup` will propagate to each individual `RadioButton`.
+
+```
+const AnswerButton = require('./AnswerButton').default
+
+
+function AnswerGroupExample() {
+  const [activity, setActivity] = React.useState('nothing')
+
+  function handleRadioChange(event) {
+    setActivity(event.target.value)
+  }
+
+  return (
+    <>
+      <AnswerGroup name="yes-no-question" onChange={handleRadioChange} >
+        <AnswerButton value="yes">Yes</AnswerButton>
+        <AnswerButton value="no">No</AnswerButton>
+      </AnswerGroup>
+      <br/>
+      <br/>
+      <br/>
+      <p>You've selected: {activity}</p>
+    </>
+  )
+};
+
+<AnswerGroupExample />
+```
+
+You can apply different button styles using the `styleType` and `successType` props
+
+```
+const AnswerButton = require('./AnswerButton').default
+
+
+function AnswerGroupExample() {
+  const [activity, setActivity] = React.useState('nothing')
+
+  function handleRadioChange(event) {
+    setActivity(event.target.value)
+  }
+
+  return (
+    <>
+      <AnswerGroup name="household-type" styleType="warning" successType="success" itemWidth="200px" onChange={handleRadioChange} >
```

Comment:
I don't think `successType` is supposed to be here...

Reply:
changed to `selectedType` throughout component